src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecteditor/view/ProjectEditor.kt

```diff
@@ -109,86 +105,64 @@ class ProjectEditor : View() {
-                            contentCard.bindClass(cardContextCssRuleProperty())
-                            contentCard.bindClass(disabledCssRuleProperty(item.first, item.second))
-                            contentCard.bindClass(hasTakesCssRuleProperty(item.second))
-                            if (item.first.value.labelKey == "chapter") {
-                                // Special rendering
-                                contentCard.titleLabel.textProperty().unbind()
-                                contentCard.titleLabel.graphic = AppStyles.chapterIcon("30px")
-                                contentCard.titleLabel.text = viewModel.activeChildProperty.value.titleKey
-                            }
-                            viewModel.contextProperty.toObservable().subscribe { context ->
-                                contentCard.actionButton.visibleProperty().bind(
-                                        viewModel.contextProperty.isEqualTo(ChapterContext.RECORD)
-                                                .or(item.second
-                                                        .and(viewModel.contextProperty.isEqualTo(ChapterContext.VIEW_TAKES)))
-                                                .or(item.first.booleanBinding { it?.selectedTake != null}
-                                                        .and(viewModel.contextProperty.isEqualTo(ChapterContext.EDIT_TAKES)))
-                                )
-                                when (context ?: ChapterContext.RECORD) {
-                                    ChapterContext.RECORD -> {
-                                        contentCard.actionButton.apply {
-                                            graphic = AppStyles.recordIcon()
-                                            text = messages["record"]
+                    chapterScrollPane = scrollpane {
+                        isFitToHeight = true
+                        isFitToWidth = true
+                        chapterFlowPane = flowpane {
+                            addClass(AppStyles.appBackground)
+                            addClass(ProjectEditorStyles.collectionsFlowpane)
+                            bindChildren(viewModel.children) {
+                                vbox {
+                                    alignment = Pos.CENTER
+                                    chaptercard(it) {
+                                        addClass(ProjectEditorStyles.collectionCard)
+                                        chapterGraphic.apply { addClass(ProjectEditorStyles.chaptercardGraphic) }
+                                        cardNumber.addClass(ProjectEditorStyles.cardNumber)
+                                        cardButton.addClass(ProjectEditorStyles.cardButton)
+                                        cardBackground.addClass(ProjectEditorStyles.cardBackground)
+                                        progressbar.addClass(ProjectEditorStyles.cardProgressbar)
```

Comment:
similar to above, setting styles here seems awkward and they probably should be style sheets for cards themselves, not this page (unless the page is explicitly overriding defaults?)?

Reply:
yes. the idea is that we are overriding the default styles for the cards here
will be important when we have to change the style/color of the cards depending on the most recently active project/chapter/verse + completed projects chapters and verses